tests/rptest/tests/pandaproxy_test.py

```diff
@@ -716,7 +716,6 @@ def test_remove_consumer_validation(self):
             })
         assert sc_res.status_code == requests.codes.no_content
 
-    @ok_to_fail  # https://github.com/redpanda-data/redpanda/issues/3454
```

Comment:
That ticket is closed as a duplicate of one that is still open:
https://github.com/redpanda-data/redpanda/issues/2501

Reply:
John are you saying that we can also close 2501 now? cc @rystsov

Reply:
The opposite: afaik the issue is not fixed.  I believe this is the one that Ben identified as a bug in the kafka client.